packages/cli/templates/platform-app-tailwind/src/App.tsx

```diff
@@ -0,0 +1,124 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { useEffect, useState } from 'react';
+import {
+    Button,
+    ButtonEmphasis,
+    Card,
+    LoadingCircle,
+    LoadingCircleSize,
+    LoadingCircleStyle,
+    Stack,
+    Text,
+} from '@frontify/fondue';
+import { PlatformAppContext, usePlatformAppBridge } from '@frontify/app-bridge';
+
+type AssetResource = {
```

Comment:
Import from the bridge?

Reply:
yes its in the bridge, but needs adjustments